### ISSUE TITLE
[BUG] Polypolish failed in the test run

### ISSUE THREAD

**Issue by wanyuac** [gbouras13/hybracter#49]
**Describe the bug**

The test run of hybracter v0.5.0 failed at the step of polypolish.

**To Reproduce**
Commands executed:
```bash
conda config --add channels defaults
conda config --add channels bioconda
conda config --add channels conda-forge
conda create -n hybracter hybracter
conda activate hybracter
hybracter -h
conda install conda-forge::mamba
hybracter install

hybracter test-hybrid --threads 32
```

Error message:
```bash
[Wed Jan 17 08:47:35 2024]
Error in rule polypolish_incomplete:
    jobid: 54
    input: hybracter_out/processing/incomplete/medaka_incomplete/Sample2/consensus.fasta, hybracter_out/processing/incomplete/bwa_incomplete/Sample2_1.sam, hybracter_out/processing/incomplete/bwa_incomplete/Sample2_2.sam
    output: hybracter_out/processing/incomplete/polypolish_incomplete/Sample2.fasta, hybracter_out/versions/Sample2/polypolish_incomplete.version
    log: hybracter_out/stderr/polypolish_incomplete/Sample2.log (check log file(s) for error details)
    conda-env: /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/c5fabf974bde0f333c1685cc1469c14a_
    shell:

        polypolish hybracter_out/processing/incomplete/medaka_incomplete/Sample2/consensus.fasta hybracter_out/processing/incomplete/bwa_incomplete/Sample2_1.sam hybracter_out/processing/incomplete/bwa_incomplete/Sample2_2.sam > hybracter_out/processing/incomplete/polypolish_incomplete/Sample2.fasta 2> hybracter_out/stderr/polypolish_incomplete/Sample2.log
        polypolish --version > hybracter_out/versions/Sample2/polypolish_incomplete.version

        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)
Logfile hybracter_out/stderr/polypolish_incomplete/Sample2.log:
================================================================================
error: unrecognized subcommand 'hybracter_out/processing/incomplete/medaka_incomplete/Sample2/consensus.fasta'

Usage: polypolish <COMMAND>

For more information, try '--help'.
================================================================================

Removing output files of failed job polypolish_incomplete since they might be corrupted:
hybracter_out/processing/incomplete/polypolish_incomplete/Sample2.fasta
Total ALE Score: -168604.508330
Output is in file: hybracter_out/processing/ale_out_files/Sample2/medaka_incomplete.ale
Closing input file
[Wed Jan 17 08:47:35 2024]
Finished job 62.
24 of 74 steps (32%) done
Removing temporary output hybracter_out/processing/ale_out_files/Sample2/medaka_incomplete.ale.
Removing temporary output hybracter_out/processing/ale_sams/Sample2_medaka_incomplete_1.sam.
[Wed Jan 17 08:47:43 2024]
Finished job 87.
25 of 74 steps (34%) done
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-01-17T084313.658883.snakemake.log
WorkflowError:
At least one job did not complete successfully.
[2024:01:17 08:47:43] ERROR: Snakemake failed
```

**Expected behavior**
I expected the pipeline to complete successfully.

**Desktop (please complete the following information):**
 - OS: Linux — Ubuntu 22.04.3 LTS
 - Conda v23.11.0

**Additional context**
I have attached the log file and a list of packages in my conda environment for your information.
[2024-01-17T084313.658883.snakemake.log](https://github.com/gbouras13/hybracter/files/13960821/2024-01-17T084313.658883.snakemake.log)
[hybracter_env_packages.txt](https://github.com/gbouras13/hybracter/files/13960832/hybracter_env_packages.txt)

In addition, here is the output of command `conda info --envs`:
```bash
hybracter             *  /home/ywan1/.conda/envs/hybracter
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/08d9e3870d1f397e2bc2f72cf1504630_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/0e1770c42a5cbb10c348ab06de0031d4_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/259b09f962750302d4bd17c5440cee7b_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/26b7aa3cf65e5d452850bb02034a890f_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/2982368a9476256dc30717256010c978_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/352ed4064ab22bf9995e86c02031e19f_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/51b5852613cda593b811b4065213fd24_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/645e771a464c722f5c39fd03ee2edca1_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/c1d130cca38fe643cfb099e9206899d6_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/c5fabf974bde0f333c1685cc1469c14a_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/d03bba3f76daff676a2be90aea903a2c_
                         /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/f6a7b3a52ed0cfbf9c8e997a26077c62_
```

**Comment by gbouras13** [comment]
Hi @wanyuac ,

This is a bug caused by polypolish's recent upgrade v0.6.0 which changed the CLI. 

I've put in a fix on the dev branch (you can see a PR) which will be available soon. Otherwise simply downgrade to v0.5.0 e.g.

```
conda activate /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/c5fabf974bde0f333c1685cc1469c14a_

mamba install polypolish==0.5.0
conda deactivate
```

George

**Comment by wanyuac** [comment]
Superb, and thank you for your prompt response! The hybracter pipeline completed successfully after downgrading polypolish following your suggestion. Happy to close this issue.

**Comment by gbouras13** [comment]
Closed by v0.6.0 merged into the main branch
